dashboards/k8s-views-global.json

```diff
@@ -1557,10 +1860,25 @@
           "editorMode": "code",
           "exemplar": true,
           "expr": "sum(node_memory_MemTotal_bytes{cluster=\"$cluster\"} - node_memory_MemAvailable_bytes{cluster=\"$cluster\"}) by (instance)",
+          "hide": false,
           "interval": "$resolution",
-          "legendFormat": "{{ node}}",
+          "legendFormat": "{{ instance }}",
```

Comment:
Because node was just wrong. The query above says `sum by (instance)` They can not be a node label. Otherwise, I have change the query to `sum by (node)`